Dockerfile

```diff
@@ -18,18 +18,21 @@ ENV SHELL /bin/bash
 
 SHELL ["/bin/bash", "-c"]
 
-# Instal rust.
+# Install rust.
 RUN sudo apt-get -y install curl clang python3 python3-pip git
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 ENV PATH="/home/erdos/.cargo/bin:${PATH}"
-RUN rustup default nightly
```

Comment:
Do we not need nightly anymore?

Reply:
Cargo reads the `rust-toolchain.toml` which ensures that ERDOS will be built with Rust nightly-2021-12-04.